logever/src/main/java/ai/smartfac/logever/entity/User.java

```diff
@@ -45,11 +47,16 @@ public class User implements UserDetails {
     private Timestamp createDt;
     @Column(name = "updated_by")
     @LastModifiedBy
+    @JsonIgnore
     private String updatedBy;
     @JsonIgnore
     @Column(name="update_dt")
     @UpdateTimestamp
     private Timestamp updateDt;
+    @Column(name = "is_active",nullable = false,columnDefinition = "BOOLEAN")
+    private Boolean isActive;
+    @Column(name = "last_login_dt")
+    private Timestamp lastLoginDt;
```

Comment:
along with this, we will also need created_at.

Reply:
Disregard, found createDt.

Reply:
removed jsonignore and createDt is now available in response.